recipes/folly/all/patches/0007-allow-builtins.patch

```diff
@@ -1,13 +1,128 @@
 diff --git a/folly/portability/Builtins.h b/folly/portability/Builtins.h
```

Comment:
```suggestion
https://github.com/facebook/folly/issues/1412
diff --git a/folly/portability/Builtins.h b/folly/portability/Builtins.h
```

Reply:
Why is my last git comment on PR "see facebook/folly#1412" not enough?

Reply:
because it's less likely someone will track down the PR in GitHub and read all of the hidden comments...

Putting it in the code will make it more likely to be seen

an idea for #3903 if it's not already there

Reply:
but this is the way to go "track down the PR/issues" and not grep the source code for comments. Nevertheless I added the cross reference to folly issue, although I still don't really like the idea to modify the git generated patches manually.

Reply:
you could have used the "patch url" instead..


https://patch-diff.githubusercontent.com/raw/conan-io/conan-center-index/pull/4048.patch it has extra information in the heading